packages/edit-site/src/components/global-styles/utils.js

```diff
@@ -1,12 +1,12 @@
 /**
  *
- * @param {string} path The variation path in the Global Styles tree.
+ * @param {string} variation The variation name.
  *
  * @return {string} The variation class name.
  */
-export function getVariationClassNameFromPath( path ) {
-	if ( ! path ) {
+export function getVariationClassName( variation ) {
+	if ( ! variation ) {
 		return '';
 	}
-	return `is-style-${ path.split( '.' )[ 1 ] }`;
+	return `is-style-variations-${ variation }`;
```

Comment:
This needs to be `is-style-${ variation }` so the classname ends up as e.g. `is-style-outline`. As it is the variations aren't coming through in the design tools previews:

<img width="277" alt="Screenshot 2023-01-27 at 11 25 47 am" src="https://user-images.githubusercontent.com/8096000/214980397-d35f709a-1a54-4331-81a2-b86ebfdec69c.png">

Going to Button > Style Variations > Outline > Typography, the preview should show the outline variation.

Reply:
Good catch, this should be fixed now.